frontend/components/TableContainer/TableContainer.tsx

```diff
@@ -82,6 +82,8 @@ interface ITableContainerProps<T = any> {
   // TODO - consolidate this functionality within `filters`
   selectedDropdownFilter?: string;
   isClientSidePagination?: boolean;
+  /** Only used on self-service page client side pagination that clicking on a action re-orders the data and we don't want that to trigger reset to page 0 */
```

Comment:
```suggestion
  /** Only used on self-service page client side pagination because clicking on a action re-orders the data and we don't want that to trigger reset to page 0 */
```